peps/pep-0738.rst

```diff
@@ -106,23 +107,25 @@ and ``x86_64`` are supported for use in emulators on development machines.
 silicon. Unlike on iOS, an ARM64 emulator uses the same ABI as a real ARM64
 device, and can run the same binaries.
 
-We propose that Tier 3 support will only cover the 64-bit platforms
+For Python 3.13 we propose that Tier 3 support will only cover the 64-bit platforms
 (``arm64-v8a`` and ``x86_64``):
 
 * ``x86`` has not been supported as a development platform since 2020, and no
   new emulator images have been released since then.
-* ``armeabi-v7a`` is still used, but its proportion of active devices is now
+* ``armeabi-v7a``'s proportion of active devices is now
   `less than 10% and steadily falling
   <https://github.com/chaquo/chaquopy/issues/709#issuecomment-1744541892>`__.
+  However, it continues to be used for watches and ultra-low-cost phones. If
+  this persists, we may need to consider adding it in a future Python version.
 
-However, no changes should be made which would impede any downstream projects
-which still wish to build for the 32-bit architectures.
+Even if 32-bit architectures are not officially supported, no changes should be
+made which would impede any downstream projects which still wish to build them.
```

Comment:
Is the reason for not including 32-bit architectures a technical limitation, or a matter of avoiding CI bloat (or both)? Dropping x86 support makes sense, as there's no devices, and no emulators - but what (if anything) would a third-party need to do to enable 32-bit ARM support?

Reply:
The main concern is whether it's feasible to run the test suite on an emulator. Performance would be significantly worse than the other architectures, because there are no "native" hosts available for the emulator. ARM64 Macs don't help here, because they don't have hardware support for ARM32 code.

For that reason, Google actually stopped releasing ARM32 emulator images in 2016 with API level 25.

I'll add these details to the PEP.